### PR TITLE
adding remarks insertion text data

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
😺
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2365)
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/724)

### CODE REVIEW

File: src/services/common.js [hunk 1]
```diff
@@ -463,9 +463,22 @@ const convertTemplateFiltersCols = (query, mapFunc) => {
   return filsCols
 }
 
+const objArrayToDictionary = (data, dictionaryKey) => {
+  // transform data from an array of objects, to an obj
```
Comment (elizabeth-jimenez) [inline src/services/common.js:467]: data: [{"a": "1"}, {"a": "2", "b": "3"}, {"a": "1", "b": "3"}]
dictionaryKey: "a"

dd: {
"1": [{"a": "1"}, {"a": "1", "b": "3"}]
"2": [{"a": "2", "b": "3"}]
}

**Comment by scott062** [comment]
- `hasMany` feedback per our convo

File: src/services/common.js (hunk 1 above)
Comment (elizabeth-jimenez) [reply src/services/common.js:467]: i'll miss you 😿

**Comment by elizabeth-jimenez** [comment]
image

**Review by scott062 — APPROVED** [review]
Looks good! 💯

**Pull request merged**